modules/ROOT/nav.adoc

```diff
@@ -1,16 +1,27 @@
 .xref:index.adoc[Anypoint Partner Manager]
 * xref:index.adoc[Overview]
 * xref:setup.adoc[Set up the Anypoint Platform for Partner Manager]
-* xref:B2B-overview.adoc[Use Anypoint Partner Manager]
+* xref:B2B-overview.adoc[Configure Anypoint Partner Manager]
  ** xref:configure-host.adoc[Configure Host]
  ** xref:configure-partner.adoc[Configure Partner]
  ** xref:x12-receive-read-settings.adoc[X12 Settings - Receive from Partner]
- ** xref:configure-message-flows.adoc[Create a Message Flow]
+ * xref:message-flows.adoc[Message Flows]
+  ** xref:configure-message-flows.adoc[Configure Message Flows]
+  ** xref:manage-message-flows.adoc[Manage Message Flows]
 * xref:x12-identity-settings.adoc[Identifiers]
 * xref:Certificates.adoc[Certificates]
 * xref:endpoints.adoc[Endpoint Types and Protocols]
-** xref:endpoint-as2-receive.adoc[AS2 Endpoint Settings]
-** xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
-** xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
+ ** xref:endpoint-as2-receive.adoc[AS2 Endpoint Settings]
+ ** xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
+ ** xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
 * xref:document-types.adoc[Message Types]
 * xref:activity-tracking.adoc[Track Activity]
+* xref:troubleshooting.adoc[Resolving Current Issues]
+ ** xref:ts-unsuccessful-deploy.adoc[Deployment is Unsuccessful]
```

Comment:
is > Is in titles
```suggestion
 ** xref:ts-unsuccessful-deploy.adoc[Deployment Is Unsuccessful]
```

Reply:
QUESTION: Can you remove deleted files from showing up in the PR as DIFFs?

Reply:
Hmm, will have to look in this.